InstanceManager.js

```diff
@@ -557,70 +651,59 @@ class InstanceManager {
 
   /// use Current/ServersRegistered to find the corresponding
   /// instance metadata
-  resolveUUID(uuid) {
+  async resolveUUID(uuid) {
     const baseUrl = endpointToUrl(this.getAgencyEndpoint());
-    return this.rpAgency({
+    const [info] = await this.rpAgency({
```

Comment:
I'd like us to be consistent about using `const` to indicate "this won't be reassigned to a different value" (but may still be mutated). You're using `const` here correctly. You should probably use `const` in the other places above too then.